### PR TITLE
 add back to top feature on how to play page 

### LINKED ISSUE #692: Add back to the top feature on How to play page
### Expected results
Back to top feature
### Actual results
No such function

### Would you like to work on this
Yes

### PR DESCRIPTION
**Pull request by PrP-11**
- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a preview link and screenshot showing after and before the changes.
- [x] Included a description of change below.
- [ ] Squashed the commits.

### Changes done in this Pull Request
- add back to top feature on how to play page 
- **Test-Link**
  http://rawgit.com/PrP-11/labyrinth/b2/howtoplay.html

 - Fixes #692

### CODE REVIEW

**Comment by PrP-11** [comment]
@niccokunzmann I think that the quality of code is fine and the reason for failure of the check is not a valid reason in javascript.

File: css/stylesheet.css [hunk 1]
```diff
@@ -210,3 +210,24 @@ img {
...
+    opacity: 0.5;
+    border: none;
+    outline: none;
+    background-color: aqua;
+    color: blue;
+    cursor: pointer;
+    padding: 15px;
+    border-radius: 10px;
+}
+#scrollToTop:hover {
```
Comment (niccokunzmann) [inline css/stylesheet.css]: Is there a reason why you duplicate the code? You can create a style sheet and include it in both documents. Or should if look differently?

File: js/contributors.js [hunk 2]
```diff
@@ -1,53 +1,66 @@
...
-      } else {
-        html += ' contributions)</p>';
-      }
-      html += '<p class="middle"><button class="button"> ';
-      html += '<a href="'+contributors.html_url+'""> View on Github';
-      html += '</button></p>';
-      html += '</a></div></div></div>';
-      $("#contributors").append(html);
+window.onload = function () {
+    $.ajax({
```
Comment (niccokunzmann) [inline js/contributors.js]: Please see the style guide. https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md#code-style-recommendations We use two spaces in all documents. Changing back and forth makes it harder to read the history and understand the change. What do you think?

Comment (PrP-11) [reply js/contributors.js]: @niccokunzmann I wrote the code with two spaces but I think my editors auto corrected it to tabspace. I will correct it. Thanks for pointing that out.

File: css/stylesheet.css (hunk 1 above)
Comment (PrP-11) [reply css/stylesheet.css]: @niccokunzmann I thought that one css file for each page should be there but I think a new stylesheet can be made. I will change the commit.

**Comment by niccokunzmann** [comment]
@PrP-11 Thanks! Somehow, there are merge conflicts. What would you like to do?

**Comment by niccokunzmann** [comment]
@PrP-11 The codacy decrese is not a problem. travis is important and the other is a feedback and allows contributors to improve the overall code quality, too.

**Comment by PrP-11** [comment]
@niccokunzmann I am working on the merge conflict. It will be corrected in a minute

**Comment by PrP-11** [comment]
@niccokunzmann I solved the conflicts and made one js file and one css file for back to top feature so that it can be used for any page in future. Now it can be happily merged.

**Comment by niccokunzmann** [comment]
Thanks!

**Comment by niccokunzmann** [comment]
Codacy is also fine ... :)

**Pull request merged**